src/applications/personalization/profile/selectors.js

```diff
@@ -80,5 +80,18 @@ export const militaryInformationLoadError = state => {
   return state.vaProfile?.militaryInformation?.serviceHistory?.error;
 };
 
-export const showNotificationSettings = state =>
-  toggleValues(state)[FEATURE_FLAG_NAMES.profileNotificationSettings];
+export const showNotificationSettings = state => {
+  const LSProfileNotificationSetting = localStorage.getItem(
+    'PROFILE_NOTIFICATION_SETTINGS', // 0 or 1
+  );
+  const FFProfileNotificationSettings = toggleValues(state)[
+    FEATURE_FLAG_NAMES.profileNotificationSettings
+  ];
+  // local setting takes precedent over FF
+  if (LSProfileNotificationSetting === 'false') {
```

Comment:
using `'true'` or `'false'` for this is totally fine, but could you update the comment on line 85?